### PR TITLE
Design improvements

### PR DESCRIPTION
**Pull request by SuhasDissa**

### CODE REVIEW

File: app/src/main/java/app/suhasdissa/vibeyou/utils/Mappers.kt [hunk 1]
```diff
@@ -78,7 +78,7 @@ val Song.asMediaItem: MediaItem
                 .setArtist(artistsText)
                 .setArtworkUri(thumbnailUri)
                 .setExtras(
-                    bundleOf("isFavourite" to isFavourite)
+                    bundleOf("isLocal" to isLocal)
```
Comment (Bnyro) [inline app/src/main/java/app/suhasdissa/vibeyou/utils/Mappers.kt:81]: Please save "isLocal" of a constant

File: app/src/main/java/app/suhasdissa/vibeyou/ui/components/ArtistCard.kt [hunk 2]
```diff
@@ -51,26 +50,30 @@ fun ArtistCard(
...
-        Spacer(Modifier.height(8.dp))
-        Text(
-            artist.artistsText,
-            style = MaterialTheme.typography.titleSmall,
-            maxLines = 1,
-            overflow = TextOverflow.Ellipsis,
-            textAlign = TextAlign.Center
-        )
-        Spacer(Modifier.height(8.dp))
+        Column(
```
Comment (Bnyro) [inline app/src/main/java/app/suhasdissa/vibeyou/ui/components/ArtistCard.kt:66]: I don't think we need this column anymore, the modifier could be passed to the text directly

**Comment by Bnyro** [comment]
Suggestion for the album screen:
1. Add icons to the buttons
2. Make the "play all" button outlined.

File: app/src/main/java/app/suhasdissa/vibeyou/ui/components/ArtistCard.kt (hunk 2 above)
Comment (SuhasDissa) [reply app/src/main/java/app/suhasdissa/vibeyou/ui/components/ArtistCard.kt:66]: I was planning to add the number of albums below the artist name. But, I couldn't figure out a way to get that data

Comment (Bnyro) [reply app/src/main/java/app/suhasdissa/vibeyou/ui/components/ArtistCard.kt:66]: Piped doesn't support it, but for local Songs it should be easy enough, shouldn't it?

**Pull request merged**